charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -422,6 +422,13 @@ manager:
       repositoryDomain: public.ecr.aws/aws-observability
       repository: adot-autoinstrumentation-node
       tag: v0.2.0
+  autoInstrumentationApplicationSignals:
```

Comment:
Given the other `autoInstrumentation` fields in here are all also for ApplicationSignals, should we just make this: 

```
autoInstrumentationRuntimeMetrics:
  java:
    enabled: "true"
```

Reply:
Or if we want to keep it more generic to support future flags `autoInstrumentationConfiguration`.

Reply:
Changed it to `autoInstrumentationConfiguration`.